nncf/common/utils/helpers.py

```diff
@@ -25,3 +25,20 @@ def create_table(header: List[str], rows: List[List[Any]]) -> str:
     :return: A string which represents a table with a header and rows.
     """
     return Texttable().header(header).add_rows(rows, header=False).draw()
+
+
+def is_accuracy_aware_training(config, compression_config_passed=False):
```

Comment:
It looks like the function is public and the user will be used in their learning pipeline. What do you think about moving it to, for example, 'nncf/config/utils.py' or 'nncf/common/helpers/training.py'?

Reply:
done, moved to `nncf/config/utils.py`